draft-ietf-rats-eat.md

```diff
@@ -1102,94 +1102,46 @@ The values for the results enumerated type are as follows:
-For example, the top-level of the token may have a UEID, a submod may have a different UEID and a further subordinate submodule may also have a UEID.
-
-The encoding of a submodule Claims-Set MUST be the same as the encoding as the token it is part of.
-
-The data type for this type of submodule is a map/object.
-It is identified when decoding by its type being a map/object.
-
-
-##### Nested Token {#Nested-Token}
-
-This type of submodule is a fully formed complete token.
-It is typically produced by a separate attester.
-It is typically used by a composite device as described in RATS Architecture {{RATS.Architecture}}
-In being a submodule of the surrounding token, it is cryptographically bound to the surrounding token.
-If it was conveyed in parallel with the surrounding token, there would be no such binding and attackers could substitute a good attestation from another device for the attestation of an errant subsystem.
-
-A nested token does not need to use the same encoding as the enclosing token.
-This is to allow composite devices to be built without regards to the encoding supported by their attesters.
-Thus, a CBOR-encoded token like a CWT can have a JWT as a nested token submodule and vice versa.
-
-
-###### Surrounding EAT is CBOR-Encoded
-
-This describes the encoding and decoding of CBOR or JSON-encoded tokens nested inside a CBOR-encoded token.
-
-If the nested token is CBOR-encoded, then it MUST be a CBOR tag and MUST be wrapped in a byte string.
-The tag identifies whether the nested token is a CWT, a CBOR-encoded detached EAT bundle, or some other CBOR-format token defined in the future.
-A nested CBOR-encoded token that is not a CBOR tag is NOT allowed.
-
-If the nested token is JSON-encoded, then the data item MUST be a text string containing JSON.
-The JSON is defined in CDDL by JSON-Nested-Token in the next section.
-
-When decoding, if a byte string is encountered, it is known to be a nested CBOR-encoded token.
-The byte string wrapping is removed.
-The type of the token is determined by the CBOR tag.
-
-When decoding, if a text string is encountered, it is known to be a JSON-encoded token.
-The two-item array is decoded and tells the type of the JSON-encoded token.
-
+~~~~CDDL
+{::include nc-cddl/detached-digest.cddl}
+~~~~
 ~~~~CDDL
 {::include nc-cddl/nested-token-cbor.cddl}
 ~~~~
+~~~~CDDL
+{::include nc-cddl/nested-token-json.cddl}
+~~~~
 
-###### Surrounding EAT is JSON-Encoded
-
-This describes the encoding and decoding of CBOR or JSON-encoded tokens nested inside a JSON-encoded token.
+When decoding a submodule in a CBOR-encoded EAT, the Claims-Set type will be encoded as a map, the Detached-Submodule-Digest type as an array, and the Nested-Token type as a CBOR-tagged object.
 
-The nested token MUST be an array of two, a text string type indicator and the actual token.
+When decoding a JSON-encoded EAT, a little more work is required because both the Nested-Token and Detached-Submodule-Digest types are arrays.
```

Comment:
Maybe go more parallel to CBOR:

When decoding a JSON-encoded EAT submodule, the Claims-Set type is encoded in a map and all the other types an array. The first member of the array gives further info. If it is an algorithm identifier it is a detached digest. If it is "JWT", "CBOR" or "BUNDLE", it is a Nested-Token in the indicated form.

Reply:
Changed to the provided text (one sentence per line).